visma-client/lib/Api/NotesV2Api.php

```diff
@@ -112,11 +112,11 @@ public function notesV2Get(ODataQueryOptions $odataQueryOptions = null)
      *
      * @throws \Struqtur\VismaEAccounting\ApiException on non-2xx response
      * @throws \InvalidArgumentException
-     * @return array of \Struqtur\VismaEAccounting\Model\NoteApi, HTTP status code, HTTP response headers (array of strings)
+     * @return array of \Struqtur\VismaEAccounting\Model\PaginatedResponseNoteApi, HTTP status code, HTTP response headers (array of strings)
```

Comment:
Nit: shouldn't it be `* @return \Struqtur\VismaEAccounting\Model\PaginatedResponseNoteApi` ?